screens/ResourceScreen.js

```diff
@@ -11,12 +11,13 @@ const ResourceScreen = () => {
   return (
     <Container>
     <Header leftButton="menu" onLeftButtonPress={() => Actions.drawerOpen()}>Resources</Header>
-
     <Content>
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
-          Resource Screen
-        </Text>
+      <View>
+        <Button variant = "secondary" style = {styles.container}>naloxone injection instructions</Button>
```

Comment:
Similarly for the other buttons. See below
```suggestion
        <Button variant="secondary" style={styles.button}>naloxone injection instructions</Button>
```